galasa-managers-parent/buildSrc/src/main/groovy/galasa.manager.ivt.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 dependencies {
-    implementation 'dev.galasa:dev.galasa:0.15.0'
+    implementation 'dev.galasa:dev.galasa:0.17.0'
```

Comment:
does this need to be bumped?

Reply:
I have changed this back, as realised the change was not necessary